src/mobs/spawn.js

```diff
@@ -62,6 +62,18 @@ export function spawn_mob(client, { mob, position, events }) {
     }),
   })
 
+  if (custom !== undefined) {
+    Object.entries(custom).forEach(custom_data => {
+      const [key, value] = custom_data
+      client.write('entity_metadata', {
+        entityId: entity_id,
+        metadata: to_metadata(minecraft_entity, {
+          [key]: value,
+        }),
+      })
+    })
```

Comment:
just forward the config
```suggestion
  client.write('entity_metadata', {
        entityId: entity_id,
        metadata: to_metadata(minecraft_entity, custom),
    })
```

Reply:
done